src/BlazorUI/Bit.BlazorUI/Components/Inputs/CircularTimePicker/BitCircularTimePicker.scss

```
@@ -234,20 +332,33 @@
     display: inline-flex;
     text-decoration: none;
     vertical-align: middle;
-    font-size: spacing(5.5);
     justify-content: center;
     -webkit-appearance: none;
     background-color: transparent;
-    color: $clr-fg-pri;
+    font-size: var(--bit-ctp-txt-fontsize);
+    // The toolbar carries the on-color of the accent it is painted with, so its buttons inherit it instead
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)